tests/conftest.py

```diff
@@ -141,29 +142,54 @@ def make_game(database, uid, players):
     return game
 
 
+def make_player(
+    state=PlayerState.IDLE,
+    global_rating=None,
+    ladder_rating=None,
+    numGames=0,
+    ladder_games=0,
+    **kwargs
+):
+    ratings = {k: v for k, v in {
+     RatingType.GLOBAL: global_rating,
+     RatingType.LADDER_1V1: ladder_rating,
+    }.items() if v is not None}
+
+    games = {k: v for k, v in {
+     RatingType.GLOBAL: numGames,
+     RatingType.LADDER_1V1: ladder_games
+    }.items() if v is not None}
```

Comment:
This `is not None` check feels slightly weird, since the default value for these is zero, not `None`.

Reply:
Yea that is odd. I don't see them set to `None` anywhere in the tests.